### ISSUE TITLE
Search Results and Pagination - 1.4.3 Contrast Minimum -  Contrast between all text in the Search results and pagination and the background is not sufficient

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9901]
As a user with low or impaired vision, I expect colour contrast between text and background colours to be sufficient that I can easily discern text.

**Linked Work Item:**
#9387 

**Expected Result:**
Contrast between all text should be at least 4.5:1.

**Actual Result:**
Contrast between text and background colours of all the text in the search results is 3.76:1.

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/f2332707-ed1d-4138-8014-06e97bab9946)
